demoConfig.csv

```diff
@@ -1,11 +1,11 @@
-DevTestLabName,ResourceGroupName,IpConfig,BastionEnabled,SharedImageGalleryName,ShutDownTime,TimezoneId,LabRegion,LabOwners,LabUsers
-Cybi1, Cybe3r,Private,true,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi2, Cybe3r,Public,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi3, Cybe3r,Shared,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi4, Cybe3r,Shared,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi5, Cybe3r,Private,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi6, Cybe3r,Private,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi7, Cybe3r,Public,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi8, Cybe3r,Shared,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi9, Cybe3r,Public,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
-Cybi10, Cybe3r1900,Shared,false,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
+DevTestLabName,ResourceGroupName,VmCreationResourceGroupName,IpConfig,BastionEnabled,SharedImageGalleryName,ShutDownTime,TimezoneId,LabRegion,LabOwners,LabUsers
+Cybi1, Cybe3r, default,Private,true,SharedImageGallery_DevTestLabs, 1900, "W. Europe Standard Time", westeurope, "phauge@microsoft.com", "lucabol@microsoft.com,phauge@microsoft.com"
```

Comment:
We should also include in the sample that we don't pass a value (should be the same as 'default') by having a double comma instead, like:
Cybi1, Cybe3r,,Private,true, ...

Reply:
Should we add this use case even if it is not a good practice?

Reply:
Yes, please - do you have any docs that point to best practices on CSV files?  Excel generates that formatting when saving as CSV automatically (double commas)